README.md

```diff
@@ -1,70 +1,50 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Calculator
+this is a web application where you have a calculator, and you can use it to solve mathmatical problems.
 
-## Available Scripts
+## Built With
+- HTML
+- CSS
+- JavaScript
+- Webpack
 
-In the project directory, you can run:
+# Live Demo
+[Demo](https://mhamad-raad.github.io/Leaderboard/)
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you change this live demo link to that of your current project.